state/sshconnrequests_test.go

```diff
@@ -155,3 +155,41 @@ func (s *SSHReqConnReqSuite) TestSSHConnReqWatcher(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	wc.AssertNoChange()
 }
+
+func (s *SSHReqConnReqSuite) TestCleanupExpiredSSHConnRequest(c *gc.C) {
+	unit := s.Factory.MakeUnit(c, nil)
+	err := s.State.InsertSSHConnRequest(
+		state.SSHConnRequestArg{
```

Comment:
create two variables holding these requests.. so that you can refer to their IDs in asserts below.